tractor/_debug.py

```diff
@@ -38,7 +37,9 @@
 _in_debug = False
 
 # lock in root actor preventing multi-access to local tty
-_debug_lock = trio.StrictFIFOLock()
+_debug_lock: trio.StrictFIFOLock = trio.StrictFIFOLock()
```

Comment:
Might pull this debugger stuff into a new PR to keep things more separate?

The main thing was being able to use the context api for tty locking which is a lot cleaner now 🏄🏼

Reply:
Nah, keeping it here for now; it's all internals anyway.